sample/interactive_sample.py

```diff
@@ -47,7 +47,8 @@
     oidc_authority=os.getenv('OIDC_AUTHORITY'),  # For External ID with custom domain
     #enable_broker_on_windows=True,  # Opted in. You will be guided to meet the prerequisites, if your app hasn't already
     #enable_broker_on_mac=True,  # Opted in. You will be guided to meet the prerequisites, if your app hasn't already
-
+    #enable_broker_on_linux=True,  # Opted in. You will be guided to meet the prerequisites, if your app hasn't already
+    
```

Comment:
```suggestion
    #enable_broker_on_linux=True,  # Opted in. You will be guided to meet the prerequisites, if your app hasn't already
```
**nit:** Remove unnecessary trailing space.
**suggestion:** Most modern editors shall support automatically trimming trailing spaces. Find that setting in your editor and change it.